community/github-cli/sources

```diff
@@ -0,0 +1 @@
+git+https://github.com/cli/cli#15a7ab6b92ec25da2d486fd235b6996857f95772
```

Comment:
Why use git for sources instead of a release tarball?

Reply:
only real reason is to avoid the `fatal: not a git repository (or any of the parent directories): .git` that pops up at compile time.

Reply:
Gotcha.